### ISSUE TITLE
[typography] global typography variables no longer work as before in V5

### ISSUE THREAD

**Issue by fetters5** [material-components/material-components-web#5621]
### Steps to reproduce
When setting global typography variables I used to be able to do the following and it worked.
```
$mdc-typography-font-family: 'Gotham SSm A', 'Gotham SSm B', arial, sans-serif;
$mdc-typography-styles-button: ( font-size: 0.95rem );
$mdc-typography-styles-body1: (font-size: 1rem ); 
$mdc-typography-styles-headline6: (font-size: 1.1rem); 

@import '~material-components-web/material-components-web';
```

However in version 5 the above does not work

I tried to use something like the following but the styles do not get updated
```
$mdc-typography-font-family: 'Gotham SSm A', 'Gotham SSm B', arial, sans-serif;
$mdc-typography-styles-button: ( font-size: 0.95rem );
$mdc-typography-styles-body1: (font-size: 1rem ); 
$mdc-typography-styles-headline6: (font-size: 1.1rem); 

@use '@material/typography' ; //or @use material-components..
```

I was able to make the font-family work by using the following

```
$gotham-font-family: unquote("'Gotham SSm A', 'Gotham SSm B', arial, sans-serif");

@use '@material/typography' with (
    $font-family: $gotham-font-family,

);
```
I do not know how to set the styles using this method however.

### Actual behavior
styles are not getting set in the material components

### Expected behavior
styles should be set for material components

### Your Environment:

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          | 5
| Browser          |
| Operating System |

**Comment by asyncLiz** [comment]
Your syntax in the last option is correct for use with sass modules. All public mixins/variables are namespaced in the [module system](https://sass-lang.com/blog/the-module-system-is-launched).

```scss
$gotham-font-family: unquote("'Gotham SSm A', 'Gotham SSm B', arial, sans-serif");

@use '@material/typography' with (
  $font-family: $gotham-font-family,
);

// Previously
// @include mdc-typography-core-styles;

// Now
@include typography.core-styles;
```

@abhiomkar is it possible to override variables in the module system when using the `@import` fallback? I imagine the [individual style variables](https://github.com/material-components/material-components-web/blob/master/packages/mdc-typography/_functions.scss#L64-L67) would still take effect, but perhaps not the [font-family](https://github.com/material-components/material-components-web/blob/master/packages/mdc-typography/_variables.scss#L27) variable.

**Comment by fetters5** [comment]
Sorry I am not following, the following will not be set, I must be missing something but I've looked through the docs and the code and its not clicking for me.  How do I get the following to take effect

```
$mdc-typography-styles-button: ( font-size: 0.95rem );
$mdc-typography-styles-body1: (font-size: 1rem ); 
$mdc-typography-styles-headline6: (font-size: 1.1rem); 
```

**Comment by jamesga1** [comment]
Having the same problem.  Used to work before updating to v5.

**Comment by asyncLiz** [comment]
So a couple of discoveries. The `material-components-web` combined package uses the module system exclusively. If you're using it, you will need to use the module system to define the font family variable;

```scss
@use '@material/typography' with (
  $font-family: Arial
);

@import 'material-components-web/material-components-web';
```

If you're using the individual packages, this is not necessary because they have backwards-compatible import files.

```scss
$mdc-typography-font-family: Arial;

@import '@material/typography/mdc-typography';
```

I would personally recommend using `npm i material-components-web` as a handy way to install everything, but then import individual packages that you use. That way you're not generating extra CSS.

The `$mdc-typography-styles-<STYLE>` global variables are not working. I'll keep investigating why and how we could fix it.

**Comment by asyncLiz** [comment]
Fixed in #5643. The [Override Styles](https://github.com/material-components/material-components-web/tree/master/packages/mdc-typography#overriding-styles) section of the readme now includes how you can customize with global variables, sass module variables, or CSS custom properties.
